README.md

```diff
@@ -126,6 +126,44 @@ If using __o-table__ in silent mode, use the mixin `oTableBase' in your table st
 }
 ```
 
+#### Themes
```

Comment:
This is a _bit_ of bikeshedding, but the more I read through this the more I wonder whether "themes" as a name is confusing. It's quite different to the current themes stuff we do.

Tell me if you've already covered this. If not, I think it's worth discussing in person (oh look, a handy meeting 🙂) but does this mean we should name the existing themes to "subthemes"? Like with o-buttons as an example, you'd have "primary" and "secondary" themes within a "masterbrand" theme.